extensions/amp-form/0.1/test/test-amp-form.js

```diff
@@ -389,8 +390,9 @@ describes.repeated('', {
         // Check xhr goes through when form is valid.
         emailInput.value = 'cool@bea.ns';
         ampForm.handleSubmitEvent_(event);
-        return timer.promise(10).then(() => {
-          expect(ampForm.xhr_.fetch).to.have.been.called;
+
+        return whenCalled(ampForm.xhr_.fetch).then(() => {
```

Comment:
the `then` part is a no-op in this case (I'd still keep it to be explicit however)